### DISCUSSION TITLE
Use mini.files as default directory explorer

### DISCUSSION THREAD

**Discussion by Dronakurl** [nvim-mini/mini.nvim#573] [Q&A] [answered]
I could not get mini.files to act as the default directory explorer.
With lazy.nvim, i installed it like this:
```lua
  {
    "echasnovski/mini.files",
    config = function()
      require("mini.files").setup({ options = { use_as_default_explorer = true } })
    end,
  },
```  
   
When I call  `:e .` to explore the local folder, I get the netrw interface.
I tried with  `oil.nvim`. Here, the plugin is called when editing a folder.
  
I tried with 0.9.4 and also with the nightly versions. Also I set up a test config, with just lazy.nvim and mini.files. Still no luck. Is there anything, I missed?

**Comment by echasnovski** [comment]
> I could not get mini.files to act as the default directory explorer. With lazy.nvim, i installed it like this:
> 
> ```lua
>   {
>     "echasnovski/mini.files",
>     config = function()
>       require("mini.files").setup({ options = { use_as_default_explorer = true } })
>     end,
>   },
> ```
> 
> When I call `:e .` to explore the local folder, I get the netrw interface. I tried with `oil.nvim`. Here, the plugin is called when editing a folder.
> 
> I tried with 0.9.4 and also with the nightly versions. Also I set up a test config, with just lazy.nvim and mini.files. Still no luck. Is there anything, I missed?

I can not reproduce and not sure why this happens. It seems that something overriding or removing this feature of 'mini.files'.

What does executing `:au MiniFiles` say?

**Reply by Dronakurl** [reply, marked as answer]
Hi, thank you for the quick reply and thank you for your awesome plugin!
It said:
```
E26: No such group or event: MiniFiles
``` 
and then it came to me: It is lazy loaded, so apparently the commands do not work yet. Setting  `lazy = false` in the  `lazy.nvim` config solves everything
